source/jormungandr/jormungandr/interfaces/v1/serializer/free_floating.py

```diff
@@ -78,9 +88,20 @@ class FreeFloatingSerializer(serpy.DictSerializer):
     distance = NestedPropertyField(attr='distance')
 
 
+class PaginationSerializer(serpy.DictSerializer):
+    start_page = NestedPropertyField(attr='start_page')
+    items_on_page = NestedPropertyField(attr='items_on_page')
+    items_per_page = NestedPropertyField(attr='items_per_page')
+    total_result = NestedPropertyField(attr='total_result')
+
+
```

Comment:
Duplicate ([Here](https://github.com/CanalTP/navitia/blob/5866b150286046604cf2795b2d9fcdf388b9a80a/source/jormungandr/jormungandr/interfaces/v1/serializer/fields.py#L166)), no ?

Reply:
The function which exists already is for protobuf.